src/store/navigationModule.ts

```diff
@@ -0,0 +1,82 @@
+import { NavStoreModule, NavState, FoodListItem } from "./navigationInterface";
+import { getFoodList } from "@/utilities/apiCalls";
+
+const navigationModule: NavStoreModule = {
+  state: () => ({
+    isFoodListAList: false,
+    fullFoodList: [],
+    filteredFoodList: [],
+    filterFoodParam: "",
+    foodListOrder: "none",
+    categoryList: [],
+    activeFoodCategory: "all"
+  }),
+  mutations: {
+    toggleListView(state: NavState) {
+      state.isFoodListAList = !state.isFoodListAList;
+    },
+    setFoodList(state: NavState, list: FoodListItem[]) {
+      state.fullFoodList = list;
+      state.filteredFoodList = state.fullFoodList;
+    },
+    orderFoodList(state: NavState) {
+      const foodList = [...state.filteredFoodList];
+      const orderList = () => {
+        switch (state.foodListOrder) {
+          case "alph_asc":
+            foodList.sort((a: any, b: any) => {
```

Comment:
avoid non-descriptive variable names, name them like firstItem, secondItem or whatever it does